orttraining/orttraining/training_api/include/onnxruntime_training_c_api.h

```diff
@@ -0,0 +1,180 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// This file contains the training c apis.
+
+#pragma once
+#include "core/session/onnxruntime_c_api.h"
+
+ORT_RUNTIME_CLASS(TrainingSession); /// Class that enables performing training for the given user models.
+ORT_RUNTIME_CLASS(CheckpointState); /// Struct that holds the training states for the training session.
```

Comment:
nit: I think we don't need to mention "Class" or "Struct" here, as it's an opaque type to the user. Maybe "Type"?